### ISSUE TITLE
Importing PySide6.QtQml results in ImportError when running frozen app

### ISSUE THREAD

**Issue by mofahead** [pyinstaller/pyinstaller#7197]
When application includes the following import:

`import PySide6.QtQml`

Running the frozen application results in the following runtime error:

`ImportError: libpyside6qml.abi3.so.6.4: cannot open shared object file: No such file or directory`

### Context information

* Output of `pyinstaller --version`: ```5.6.1```
* Version of Python: `3.10.6 `
* Platform: `Ubuntu 22.04.1, 5.15.0-52-generic `
* How you installed Python: `apt (included in distro`
* Did you also try this on another platform? `No`
* Latest dev version: `Yes, downloaded and tried today, same result`

### Make sure [everything is packaged correctly](https://github.com/pyinstaller/pyinstaller/wiki/How-to-Report-Bugs#make-sure-everything-is-packaged-correctly)

  * [x] start with clean installation
  * [x] use the latest development version
  * [x] Run your frozen program **from a command window (shell)** — instead of double-clicking on it
  * [x] Package your program in **--onedir mode**
  * [x] Package **without UPX**, say: use the option `--noupx` or set `upx=False` in your .spec-file
  * [x] Repackage you application in **verbose/debug mode**. For this, pass the option `--debug` to `pyi-makespec` or `pyinstaller` or use `EXE(..., debug=1, ...)` in your .spec file.

### A minimal example program which shows the error

```
import PySide6.QtQml   # just this one line is needed to reproduce
```

### Stacktrace / full error message

```
import _frozen_importlib # frozen
import _imp # builtin
import '_thread' # <class '_frozen_importlib.BuiltinImporter'>
import '_warnings' # <class '_frozen_importlib.BuiltinImporter'>
import '_weakref' # <class '_frozen_importlib.BuiltinImporter'>
import '_io' # <class '_frozen_importlib.BuiltinImporter'>
import 'marshal' # <class '_frozen_importlib.BuiltinImporter'>
import 'posix' # <class '_frozen_importlib.BuiltinImporter'>
import '_frozen_importlib_external' # <class '_frozen_importlib.FrozenImporter'>
# installing zipimport hook
import 'time' # <class '_frozen_importlib.BuiltinImporter'>
import 'zipimport' # <class '_frozen_importlib.FrozenImporter'>
# installed zipimport hook
# zipimport: found 161 names in '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip'
import '_codecs' # <class '_frozen_importlib.BuiltinImporter'>
import 'codecs' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'encodings.aliases' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/encodings/">
import 'encodings' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'encodings.utf_8' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/encodings/">
import '_signal' # <class '_frozen_importlib.BuiltinImporter'>
import '_abc' # <class '_frozen_importlib.BuiltinImporter'>
import 'abc' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'io' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import '_struct' # <class '_frozen_importlib.BuiltinImporter'>
import 'zlib' # <class '_frozen_importlib.BuiltinImporter'>
import '_stat' # <class '_frozen_importlib.BuiltinImporter'>
import 'stat' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import '_collections_abc' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'genericpath' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'posixpath' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'os' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'types' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'enum' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import '_sre' # <class '_frozen_importlib.BuiltinImporter'>
import 'sre_constants' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'sre_parse' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'sre_compile' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'itertools' # <class '_frozen_importlib.BuiltinImporter'>
import 'keyword' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import '_operator' # <class '_frozen_importlib.BuiltinImporter'>
import 'operator' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'reprlib' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import '_collections' # <class '_frozen_importlib.BuiltinImporter'>
import 'collections' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import '_functools' # <class '_frozen_importlib.BuiltinImporter'>
import 'functools' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import '_locale' # <class '_frozen_importlib.BuiltinImporter'>
import 'copyreg' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 're' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'fnmatch' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'ntpath' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'warnings' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'errno' # <class '_frozen_importlib.BuiltinImporter'>
import 'urllib' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'urllib.parse' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/urllib/">
import 'pathlib' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'token' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'tokenize' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
# PyInstaller: FrozenImporter(/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/test?65843)
# ctypes not found in PYZ
import inspect # PyInstaller PYZ
import ast # PyInstaller PYZ
import '_ast' # <class '_frozen_importlib.BuiltinImporter'>
import contextlib # PyInstaller PYZ
import 'contextlib' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'ast' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import dis # PyInstaller PYZ
import opcode # PyInstaller PYZ
# _opcode not found in PYZ
# extension module '_opcode' loaded from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/lib-dynload/_opcode.cpython-310-x86_64-linux-gnu.so'
# extension module '_opcode' executed from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/lib-dynload/_opcode.cpython-310-x86_64-linux-gnu.so'
import '_opcode' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7fb4a921a290>
import 'opcode' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'dis' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# collections.abc not found in PYZ
import 'collections.abc' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/collections/">
import importlib # PyInstaller PYZ
import 'importlib' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import importlib.machinery as importlib.machinery # PyInstaller PYZ (__path__ override: importlib)
import 'importlib.machinery' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# linecache not found in PYZ
import 'linecache' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'inspect' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import subprocess # PyInstaller PYZ
import signal # PyInstaller PYZ
import 'signal' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import threading # PyInstaller PYZ
# _weakrefset not found in PYZ
import '_weakrefset' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'threading' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'fcntl' # <class '_frozen_importlib.BuiltinImporter'>
# msvcrt not found in PYZ
import '_posixsubprocess' # <class '_frozen_importlib.BuiltinImporter'>
import 'select' # <class '_frozen_importlib.BuiltinImporter'>
import selectors # PyInstaller PYZ
import 'math' # <class '_frozen_importlib.BuiltinImporter'>
import 'selectors' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'subprocess' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import PySide6 # PyInstaller PYZ
import textwrap # PyInstaller PYZ
import 'textwrap' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import shiboken6 # PyInstaller PYZ
import zipfile # PyInstaller PYZ
import 'binascii' # <class '_frozen_importlib.BuiltinImporter'>
import importlib.util as importlib.util # PyInstaller PYZ (__path__ override: importlib)
import importlib._abc as importlib._abc # PyInstaller PYZ (__path__ override: importlib)
import 'importlib._abc' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'importlib.util' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import shutil # PyInstaller PYZ
import bz2 # PyInstaller PYZ
import _compression # PyInstaller PYZ
import '_compression' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# _bz2 not found in PYZ
# extension module '_bz2' loaded from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/lib-dynload/_bz2.cpython-310-x86_64-linux-gnu.so'
# extension module '_bz2' executed from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/lib-dynload/_bz2.cpython-310-x86_64-linux-gnu.so'
import '_bz2' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7fb4a92d2e60>
import 'bz2' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import lzma # PyInstaller PYZ
# _lzma not found in PYZ
# extension module '_lzma' loaded from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/lib-dynload/_lzma.cpython-310-x86_64-linux-gnu.so'
# extension module '_lzma' executed from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/lib-dynload/_lzma.cpython-310-x86_64-linux-gnu.so'
import '_lzma' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7fb4a92d34c0>
import 'lzma' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'shutil' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'zipfile' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import base64 # PyInstaller PYZ
import 'base64' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# traceback not found in PYZ
import 'traceback' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import tempfile # PyInstaller PYZ
import random # PyInstaller PYZ
import bisect # PyInstaller PYZ
import '_bisect' # <class '_frozen_importlib.BuiltinImporter'>
import 'bisect' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import '_random' # <class '_frozen_importlib.BuiltinImporter'>
import '_sha512' # <class '_frozen_importlib.BuiltinImporter'>
import 'random' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# weakref not found in PYZ
import 'weakref' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/">
import 'tempfile' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import typing # PyInstaller PYZ
import 'typing' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# shiboken6.Shiboken not found in PYZ
# numpy not found in PYZ
import 'xxsubtype' # <class '_frozen_importlib.BuiltinImporter'>
# encodings.cp437 not found in PYZ
import 'encodings.cp437' # <zipimporter object "/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/base_library.zip/encodings/">
import 'shibokensupport' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.feature' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature.mapping' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature.errorhandler' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature.layout' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature.lib' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature.lib.tool' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature.parser' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
# PySide6.support not found in PYZ
import 'shibokensupport.signature.importhandler' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import 'shibokensupport.signature.lib.enum_sig' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
import argparse # PyInstaller PYZ
import gettext # PyInstaller PYZ
import 'gettext' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'argparse' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import logging # PyInstaller PYZ
import string # PyInstaller PYZ
import '_string' # <class '_frozen_importlib.BuiltinImporter'>
import 'string' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'atexit' # <class '_frozen_importlib.BuiltinImporter'>
import 'logging' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# shiboken6.Shiboken not found in PYZ
# extension module 'shiboken6.Shiboken' loaded from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/shiboken6/Shiboken.abi3.so'
# extension module 'shiboken6.Shiboken' executed from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/shiboken6/Shiboken.abi3.so'
import 'shiboken6.Shiboken' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7fb4a9179a20>
import 'shibokensupport.signature.lib.pyi_generator' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
# PySide6.support not found in PYZ
import 'shibokensupport.signature.loader' # <signature_bootstrap.EmbeddableZipImporter object at 0x7fb4a9162230>
# extension module 'shiboken6.Shiboken' loaded from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/shiboken6/Shiboken.abi3.so'
# extension module 'shiboken6.Shiboken' executed from '/home/automator/sandbox/pyinstaller-pyside6-testcase/dist/test/shiboken6/Shiboken.abi3.so'
import 'shiboken6.Shiboken' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7fb4a90feaa0>
import 'shiboken6' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
import 'PySide6' # <pyimod02_importers.FrozenImporter object at 0x7fb4a93dc640>
# PySide6.QtQml not found in PYZ
Traceback (most recent call last):
  File "test.py", line 1, in <module>
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 674, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 571, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 1176, in create_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
ImportError: libpyside6qml.abi3.so.6.4: cannot open shared object file: No such file or directory
[104115] Failed to execute script 'test' due to unhandled exception!
# (snip)
```

**Comment by rokm** [comment]
Hmmm, indeed. I cannot reproduce this on my main Fedora 36 system, nor under Ubuntu 20.04 (which is also what our CI is running, and the QtQml tests are passing there), but I can reproduce it under Ubuntu 22.04. For some reason `libpyside6qml.abi3.so.6.4` is not collected there - will check why this is the case.

**Comment by rokm** [comment]
The problem is that `ldd` seems to come up empty for certain PySide6 extensions, QtQml among them:
```
ldd venv/lib/python3.10/site-packages/PySide6/QtQml.abi3.so
```
And it looks like it comes up empty because it actually crashes. So that's not our bug - if we cannot use `ldd` to analyze binary dependencies, shared libraries are expected to end up missing.

The interesting part is that this does not happen on Fedora 36 that also uses glibc 2.35. And more interestingly, it also does not seem to happen in `ubunut:jammy` Docker container, while it does in 22.04 desktop installation in VirtualBox VM (perhaps the container is missing some library whose resolution would trigger the crash in `ldd`).

**Comment by bwoodsend** [comment]
How much damage do you think it would do if we made non-zero exit codes from `ldd` fatal?

https://github.com/pyinstaller/pyinstaller/blob/93446fffd58ae33992031902607daa6a0ffc1996/PyInstaller/depend/bindepend.py#L615

**Comment by rokm** [comment]
Not sure, we'd need to check what are situations that make `ldd` return a non-zero exit code. For example, I've seen it print warnings about not having executable permissions on the shared lib (which I suppose is fair and true), but still print the imports; we wouldn't want to fail in such cases.

Although in this particular case, since the process segfaults, its exit code will be invalid anyway (and there should be some other parameter indicating that it crashed).

**Comment by lmdu** [comment]
I have the same issue. On Ubuntu 22.04, we used pyinstaller 5.5, 5.6, 5.6.1 to build executable with PySide6.2.4, both libQt6Gui and libQt6Widgets library file are missing. However, these versions work well on Ubuntu 20.04.

**Comment by bwoodsend** [comment]
I guess we'll need to report this on Ubuntu's launchpad if we want it to change. It looks like they're currently unaware of it ([search query](https://bugs.launchpad.net/ubuntu?field.searchtext=ldd+crash&search=Search&field.status%3Alist=NEW&field.status%3Alist=INCOMPLETE_WITH_RESPONSE&field.status%3Alist=INCOMPLETE_WITHOUT_RESPONSE&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&field.status%3Alist=INPROGRESS&field.status%3Alist=FIXCOMMITTED&field.assignee=&field.bug_reporter=&field.omit_dupes=on&field.has_patch=&field.has_no_package=)).

**Comment by rokm** [comment]
https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1995362

This will become a hassle for us [once the CI runners switch to `ubuntu-22.04`](https://github.com/rokm/pyinstaller/actions/runs/3347474791).

**Comment by bwoodsend** [comment]
For what it's worth, latest Ubuntu is a poor choice for a build platform since its GLIBC version will likely be higher than that of whoever tries to run your application which ultimately leads to symbol errors at runtime. Stick to an older version for building.

**Comment by lmdu** [comment]
Indeed, Ubuntu 22.04 is not a good choice. But, my executable built on Ubuntu 20.04 causes an error when running on Ubuntu 22.04.

`symbol lookup error: /lib/x86_64-linux-gnu/libEGL_mesa.so.0: undefined symbol: wl_proxy_marshal_flags`

**Comment by mofahead** [comment]
I’m not sure if this is recommended by the pyinstaller team but you may want to look into excluding that library from the build. 

There’s an interesting list of [recommended libraries to exclude](https://github.com/AppImageCommunity/pkg2appimage/blob/master/excludelist) in the AppImage repo.

**Comment by mofahead** [comment]
For my specific use case, I am targeting Ubuntu 22.04 only, so prefer to build there for maximum compat. My workaround was to switch to PyQt6 which happily required only trivial code changes.

**Comment by bwoodsend** [comment]
It looks like launchpad uses some kind of _heat_ metrics to prioritise issues. Hitting the me too button (https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1995362/+affectsmetoo) increases the priority. Do me a favour and press that button.

**Comment by rokm** [comment]
So the bad news is that this is not specific to Ubuntu, but rather to (vanilla) `glibc` 2.35. For distributions based on this version of `glibc`, the behavior depends on whether they apply fix from https://github.com/bminor/glibc/commit/10fe3cd309b32c003a6b98e08928e7d6007caecf in their package or not. For example, Fedora 36 does, but Ubuntu 22.04 currently does not.

I suppose the silver lining is that if we are forced to switch to 22.04 for our CI runners, we can set up a PPA and install a patched version of `glibc` from there...

**Comment by bwoodsend** [comment]
I think I'd be more inclined to just run the whole job in a docker container if it really came to that.

**Comment by sofialoveeef** [comment]
have you tried:
```
sudo find / -name libpys*
```
find libpyside6qml.abi3.so.6.4 path, and create a soft link 
```
 sudo ln -s $path /lib
```

**Comment by quentin-cy** [comment]
> have you tried:
> 
> ```
> sudo find / -name libpys*
> ```
> 
> find libpyside6qml.abi3.so.6.4 path, and create a soft link
> 
> ```
>  sudo ln -s $path /lib
> ```

Thank you for that workaround, ~~it allowed us to "fix" our Ubuntu 22.04 runner.~~ (It only fixes execution if you have the lib installed somewhere, not the packing process)
Also one interesting point from our side. 
We encountered this problem after upgrading from 'PySide6==6.2.4' to 'PySide6~=6.3.2'. The executable was working fine before. Looks like something changed on pyside side :)

**Comment by rokm** [comment]
Fixed by [`libc6` `2.35-0ubuntu3.3`](https://packages.ubuntu.com/jammy-updates/libc6).
